apps/server/src/test-adapter/adapter.test.ts

```diff
@@ -1,257 +1,261 @@
-import { ConnectionStatus } from '@repo/utils'
-import { TestAdapter } from './adapter'
-import { testExampleInstitution } from './constants'
-import { MappedJobTypes } from '../shared/contract'
+import { ConnectionStatus } from "@repo/utils";
+import { TestAdapter } from "./adapter";
+import { testExampleInstitution } from "./constants";
+import { MappedJobTypes } from "../shared/contract";
 
-const labelText = 'testLabelText'
-const aggregator = 'aggregator'
+const labelText = "testLabelText";
+const aggregator = "aggregator";
 
 const testAdapter = new TestAdapter({
   labelText,
-  aggregator
-})
+  aggregator,
+});
+
+jest.mock("../services/storageClient/redis");
 
-jest.mock('../services/storageClient/redis')
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I would argue that including a file-wide disable is not a good pattern in most cases, as it would keep a user from knowing when they introduced `any` going forward in new code, when they may not mean to. 

Using `eslint-disable-next-line` feels like a better pattern, because new code will go through the normal linting process.

Reply:
I guess in a test file it's less of a concern.

Reply:
Yeah I wouldn't have done this if it wasn't in a test file.